lesson28/src/js/register.js

```diff
@@ -111,5 +110,17 @@ const saveUserData = () => {
         password : password.value,
     }
 
-    window.localStorage.setItem("userData", JSON.stringify(userData));
+    if( isRegisteredEmail(email) ){
+        validation.showErrorMessage(email,"登録済みのメールアドレスです");
+    }else{
+        window.localStorage.setItem("userData", JSON.stringify(userData));
+        window.location.href = "register-done.html";
+    }
+}
+
+const isRegisteredEmail = (email) => {
+    const userData = JSON.parse(window.localStorage.getItem("userData"));
```

Comment:
Q.
Is "window" necessary?
I thought `localStorage.getItem("userData")` was sufficient.
[ref](https://developer.mozilla.org/ja/docs/Web/API/Storage/getItem)

Reply:
Exactlly, the `window` could have been omitted!
This is because localStrage is a property that is automatically set on the `window` object, the global object of the browser. 
I fixed.
https://github.com/nakagawamai/js-lesson/pull/39/commits/8e5616131f52366d9d8c21af9522aff8228e91e0